fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastore.java

```diff
@@ -231,6 +231,16 @@ public HivePartitionStats getTableStatistics(String dbName, String tblName) {
                 .map(FieldSchema::getName)
                 .collect(toImmutableList());
         List<ColumnStatisticsObj> statisticsObjs = client.getTableColumnStats(dbName, tblName, dataColumns);
+        if (statisticsObjs.isEmpty()) {
+            // Try to use spark unpartitioned table column stats
+            try {
+                if (table.getParameters().keySet().stream().anyMatch(k -> k.startsWith("spark.sql.statistics.colStats."))) {
```

Comment:
nit: I think it's best to add session variable for control and the default value is true. So if there are any issues in user env , we can disable the use of spark stats

Reply:
At present, we can't get `SessionVariable` from  `HiveMetastore.java`, so session variable is not easy to be added.
I add the variable in `Config.java` instead.